pkgs/applications/networking/cluster/atlantis/default.nix

```
@@ -2,16 +2,18 @@
 
 buildGoModule rec {
   pname = "atlantis";
-  version = "0.22.3";
+  version = "0.27.0";
 
   src = fetchFromGitHub {
     owner = "runatlantis";
     repo = "atlantis";
     rev = "v${version}";
-    sha256 = "sha256-A/FT9t5Z+Iw1mVwS3d5Cc86A9e6jVbEtmEWroVUhhtw=";
+    hash = "sha256-a+xrmEHkSh5kicxIIxnoXgF9ep2ay5kCXwMR2sAVJIA=";
   };
+  date = "1970-01-01T00:00:00Z";
+  ldflags = [ "-X=main.version=${version}" "-X=main.date=${date}"];
```

```suggestion
  ldflags = [
    "-X=main.version=${version}"
    "-X=main.date=1970-01-01T00:00:00Z"
  ];
```

There is no need to add `date` as an attr to the derivation